test/functional/tests/common/lvm_common_utils.py

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright(c) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import datetime
+
+from storage_devices.lvm import get_block_devices_list
+
+from api.cas.init_config import InitConfig
+from test_tools import fs_utils
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import IoEngine, ReadWrite, VerifyMethod
+from test_utils.size import Size, Unit
+
+opencas_conf_path = "/etc/opencas/opencas.conf"
```

Comment:
use `from api.cas.init_config import opencas_conf_path` instead - keep the path in one place